### PR TITLE
REL-2792: skip header parsing

### PR DESCRIPTION
**Pull request by swalker2m**
The issue in question is that we don't know how to interpret a horizons query response header in the case of Saturn's moon Albiorix.  The header in question is

````
*******************************************************************************
 Revised: Aug 11, 2014               Albiorix / (Saturn)                    626

 For MEAN orbital elements, see  http://ssd.jpl.nasa.gov/sat_elem.html
 Recently discovered Saturnian irregular.
*******************************************************************************
````

which doesn't correspond to anything we are expecting.  This PR introduces an idea for how to handle these situations: skip the header processing when we only care about the ephemeris anyway.

I'm not sure if there are other ramifications of this change, but it does allow us to work with Albiorix and comet Schaumasse...

### CODE REVIEW

File: bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala [hunk 1]
```diff
@@ -160,8 +162,15 @@ object HorizonsService2 {
...
 
+    val replyType = target match {
+      case Comet(des)            => HorizonsReply.ReplyType.COMET
+      case AsteroidNewStyle(des) => HorizonsReply.ReplyType.MINOR_OBJECT
+      case AsteroidOldStyle(num) => HorizonsReply.ReplyType.MINOR_OBJECT
+      case MajorBody(num)        => HorizonsReply.ReplyType.MAJOR_PLANET
+    }
+
     def buildEphemeris(m: GetMethod): IO[Long ==>> E]  =
-      IO(CgiReplyBuilder.buildResponse(m.getResponseBodyAsStream, m.getRequestCharSet)).map(toEphemeris)
```
Comment (cquiroz) [inline bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala]: is `buildResponse` not used anymore then?

Comment (swalker2m) [inline bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala]: It appears that the PIT still uses it, via the `CgiBuilder.execute` method.

File: bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala [hunk 2]
```diff
@@ -160,8 +162,15 @@ object HorizonsService2 {
         TIME_DIGITS      -> FRACTIONAL_SEC
       )
 
+    val replyType = target match {
+      case Comet(des)            => HorizonsReply.ReplyType.COMET
+      case AsteroidNewStyle(des) => HorizonsReply.ReplyType.MINOR_OBJECT
+      case AsteroidOldStyle(num) => HorizonsReply.ReplyType.MINOR_OBJECT
+      case MajorBody(num)        => HorizonsReply.ReplyType.MAJOR_PLANET
+    }
```
Comment (swalker2m) [inline bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala]: I'm not sure whether this is the correct mapping, or if there is a correct mapping.  It appears to be irrelevant since we just ignore everything but the ephemeris data in the `HorizonsReply` we get back from `CgiReplyBuilder`.

File: bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala (hunk 1 above)
Comment (cquiroz) [inline bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala]: I guess we'll need to update the PIT when the time comes

**Comment by cquiroz** [comment]
Looks good to me :+1:

File: bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala (hunk 2 above)
Comment (tpolecat) [inline bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala]: This is correct, modulo nothing in HORIZONS ever actually being correct.

**Comment by tpolecat** [comment]
Thanks! 👍
